subsys/bluetooth/Kconfig

```diff
@@ -131,6 +131,26 @@ config BT_EXT_ADV_MAX_ADV_SET
 	  Maximum number of simultaneous Bluetooth advertising sets
 	  supported.
 
+config BT_PER_ADV
+	bool "Periodic Advertising and Scanning support [EXPERIMENTAL]"
+	help
+	  Select this to enable Periodic Advertising API support.
```

Comment:
Could have a description of what this feature is.

Reply:
I attempted to write some description. Let me know if you have any comments on the text.

Reply:
Looks good. Thank you.